doc/manual/manual.md

```diff
@@ -2621,6 +2622,82 @@ the value of an array element.
 * `jvm_field_is : JVMValue -> String -> JVMValue -> JVMSetup ()`
 specifies the name of an object field.
 
+In the experimental MIR verification implementation, the following functions
```

Comment:
Please check if the documentation here makes sense, since MIR structs require a more careful explanation than LLVM structs.